style.css

```diff
@@ -0,0 +1,1086 @@
+  justify-content: space-between;
+  align-items: center;
+}
+
+.latest-funding .section-header-text h5 {
+  color: #f42;
+  font-size: 1.5em;
+}
+
+.latest-funding .section-header-text a {
+  text-decoration: none;
+}
+
+.latest-section-wrapper .header-link a {
+  text-decoration: none;
+  color: #f42;
+}
+
+.latest-funding .section-header-text .links a {
+  margin-left: 10px;
+  text-decoration: none;
+  color: #f42;
+}
+
+.latest-funding .cards-wrapper {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+}
+
+.latest-funding .cards-wrapper .cards {
+  width: 23%;
+  height: 270px;
+}
+
+.latest-funding .cards-wrapper .cards a {
+  text-decoration: none;
+}
+
+.latest-section-wrapper > div > div img {
+  width: 100%;
+  height: 200px;
+}
+
+.latest-funding .cards-wrapper > div img {
+  width: 20%;
+  height: 50px;
+  margin: 10px 10px;
+}
+
+.latest-funding .cards-wrapper .cards .top-section {
+  width: 100%;
+  background-color: #f42;
+  height: 40px;
+}
+
+.latest-funding .cards-wrapper .cards .down-section {
+  width: 100%;
+  background-color: rgb(255, 233, 230);
+  height: 70px;
+}
+
+.latest-funding .cards-wrapper .cards .top-section .head-text {
+  display: flex;
+}
+
+.latest-funding .cards-wrapper .top-section .head-text > div h4 {
+  margin: 0;
+  padding: 0;
+}
+
+.blogs-wrapper .blogs-card .inner-img p {
+  width: 70%;
+  color: #f42;
+  font-weight: 800;
+  font-size: 0.75em;
+}
+
+.blogs-wrapper .blogs-card .inner-img p span {
+  font-weight: 600;
+  font-size: 0.8em;
+}
+
+.latest-funding .cards-wrapper .cards .tags p {
+  font-size: 0.8em;
+}
+
+.latest-funding .cards-wrapper .cards .tags p i {
+  font-size: 1em;
+  margin-right: 10px;
+}
+
+.latest-funding .cards-wrapper .cards .main-card-text p {
+  font-size: 0.8em;
+  color: gray;
+}
+
+.latest-funding .cards-wrapper .top-section .head-text > div p {
+  margin: 0 10px;
+  padding: 0;
+  font-size: 0.8em;
+  color: gray;
+}
+
+.latest-funding .cards-wrapper .top-section .head-text > div .title {
+  color: white;
+  font-weight: 700;
+  margin: 10px 10px;
+}
+
+.latest-funding .cards-wrapper .top-section .head-text > div .money {
+  margin: 10px 10px;
+  color: black;
+  margin-top: 20px;
+}
+
+.latest-funding .cards-wrapper .cards .tags {
+  margin: 0 10px;
+}
+
+.latest-funding .cards-wrapper .cards .main-card-text {
+  margin: 0 15px;
+}
+
+/* blogs section */
+.blogs-wrapper {
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  width: 85vw;
+  margin: 0 auto;
+}
+
+.blogs-wrapper > div {
+  width: 32%;
+}
+
+.blogs-wrapper .top-big-link {
+  text-decoration: none;
+  color: #f42;
+  font-weight: 800;
+  font-size: 1.3em;
+}
+
+.blogs-wrapper > div .img-wrapper img {
+  width: 32%;
+}
+
+.blogs-wrapper .blogs-card .card-links ul li {
+  padding: 5px 0;
+  margin: 5px 0;
+  border-bottom: 1px solid lightgray;
+}
+
+.blogs-wrapper .blogs-card .card-links ul {
+  margin: 0;
+  padding: 0;
+  margin-left: 17px;
+}
```

Comment:
## Shorthand Properties Rule[1]
>Shorthand properties allow you to set the rules of numerous CSS properties simultaneously. Using a shorthand property will shrink your CSS and increase readability.

For instance, instead of using margin-top, margin-right, margin-bottom, margin-left separately you can simply use just one line.
 
 - [ ] Make in just one border line 
 ```css
  margin: 0 0 0 17px;
 ```
<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_